docs/agent_setup.md

```diff
@@ -0,0 +1,27 @@
+# Agent Setup
+
+Installing the DCI agent is relatively straightforward. We have a role which should do the trick in [dci-agent-setup](/dci-agent-setup).
```

Comment:
`An Ansible role has been created to install [dci-agent-setup](/dci-agent-setup).`